test/application-test.ts

```diff
@@ -9,3 +10,14 @@ test('can be instantiated', function(assert) {
   let app = new Application({ rootName: 'app', resolver: new BlankResolver });
   assert.ok(app, 'app exists');
 });
+
+test('accepts options for rootName, resolver and document', function(assert) {
+  const resolver = new BlankResolver;
+  let app = new Application({ rootName: 'app', resolver });
+  assert.equal(app.rootName, 'app');
+  assert.equal(app.resolver, resolver);
+  assert.equal(app.document, window.document, 'defaults to window document if document is not provided in options');
+  let customDocument = <Simple.Document>{};
```

Comment:
I wonder if we can actually bring in `simple-dom` and test rendering here in a test?

It _seems_ like it should work, and it would be nice to confirm it does via a test (even though the tests will still be running in the browser)...